builder/dockerfile/instructions/commands.go

```diff
@@ -110,6 +110,25 @@ type MaintainerCommand struct {
 	Maintainer string
 }
 
+// NewLabelCommand creates a new 'LABEL' command
+func NewLabelCommand(k string, v string, expand bool) *LabelCommand {
+	// Value must be single quoted to prevent env variable expansion if not expected
+	if !expand {
```

Comment:
What I meant for this parameter was to set it as a property of `LabelCommand`, like `& LabelCommand{expand: expand, ...}` and then change the expand command to:

```
 func (c *LabelCommand) Expand(expander SingleWordExpander) error {
        if !c.expand {
           return nil
        }
 	return expandKvpsInPlace(c.Labels, expander)
 }
```

Reply:
Ah, my fault to misunderstand that,  I'll add the `expand` as property to the `LabelCommand` and leave the expand decision to the `dispatch` component according to the property value. Nice 😄